CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 * [FEATURE] OTLP receiver: Add new option `otlp.promote_resource_attributes`, for any OTel resource attributes that should be promoted to metric labels. #14200
 * [ENHANCEMENT] OTLP receiver: Warn when encountering exponential histograms with zero count and non-zero sum. #14706
+* [ENHANCEMENT] Storage: Implement limit in mergeGenericQuerier. #14489
```

Comment:
Changelog entries need to be understandable by administrators of Prometheus, so it's important to focus on the (visible) effects and not on the implementation.